backend/src/main/java/org/eventplanner/events/rest/dto/UpdateEventRequest.java

```diff
@@ -11,25 +11,27 @@
 import static org.eventplanner.common.ObjectUtils.mapNullable;
 
 public record UpdateEventRequest(
-    @Nullable String name,
-    @Nullable String state,
-    @Nullable String note,
-    @Nullable String description,
-    @Nullable String start,
-    @Nullable String end,
-    @Nullable List<LocationRepresentation> locations,
-    @Nullable List<SlotRepresentation> slots
+        @Nullable String name,
+        @Nullable String state,
+        @Nullable String note,
+        @Nullable String description,
+        @Nullable String start,
+        @Nullable String end,
+        @Nullable List<LocationRepresentation> locations,
+        @Nullable List<SlotRepresentation> slots,
+        @Nullable List<RegistrationRepresentation> registrations
```

Comment:
registrations should not be updated together with the event to prevent conflicts (users also write to the registrations)